### PR TITLE
Site Editor: Update preloaded paths

### PR DESCRIPTION
**Pull request by Mamaduka**
> Permissions for creating pages (`OPTIONS /wp/v2/pages`). Both requests are made in the `useBlockEditorSettings` hook, at the top of the post editor React tree.

Matches the preloaded path for the posts editor and hopefully will improve the Site Editor loading time a bit.

Gutenberg Issue: https://github.com/WordPress/gutenberg/pull/64890
Trac ticket: https://core.trac.wordpress.org/ticket/61884

### CODE REVIEW

File: src/wp-admin/site-editor.php
Comment (Mamaduka) [inline src/wp-admin/site-editor.php:91]: Avoids hardcoding paths. Matches the post editor.

https://github.com/WordPress/wordpress-develop/blob/b20e852a56c0970684a44166fcffa3d041f05eb8/src/wp-admin/edit-form-blocks.php#L61

**Comment by noisysocks** [comment]
@Mamaduka: Thanks for the PR! Please add a Trac ticket and put it in the 6.7 milestone so that folks will find this and review it ❤️

**Comment by Mamaduka** [comment]
@noisysocks, I re-opened a similar ticket I had for 6.7.

**Comment by Mamaduka** [comment]
@noisysocks, does this need another review/approval? It has been approved by @gziolo in the Gutenberg repo.

**Comment by noisysocks** [comment]
You're a core committer so technically you can commit without review 😛 Best practice is to get a second pair of eyes on it though.

**Review by noisysocks — APPROVED** [review]
It looks good to me 👍

**Comment by Mamaduka** [comment]
Committed in https://core.trac.wordpress.org/changeset/58984.

**Pull request closed without merging**
